addons/web/static/src/scss/fields.scss

```diff
@@ -411,6 +411,18 @@
         }
     }
 
+    // Signature
+    .o_signature {
+        outline: 1px solid lightgrey;
```

Comment:
try to not hardcode colors in the styles, make them depend on an existing variable